Cargo.toml

```diff
@@ -25,17 +25,17 @@ path = "bin/submit_transaction.rs"
 integration = []
 
 [dependencies]
-init4-bin-base = { version = "0.3.4", features = ["perms"] }
+init4-bin-base = { version = "0.4.1", features = ["perms"] }
 
-signet-constants = { git = "https://github.com/init4tech/signet-sdk", rev = "b8251ff0fec7cb14ca87e6f95c14f56bc2593049" }
-signet-sim = { git = "https://github.com/init4tech/signet-sdk", rev = "b8251ff0fec7cb14ca87e6f95c14f56bc2593049" }
-signet-tx-cache = { git = "https://github.com/init4tech/signet-sdk", rev = "b8251ff0fec7cb14ca87e6f95c14f56bc2593049" }
-signet-types = { git = "https://github.com/init4tech/signet-sdk", rev = "b8251ff0fec7cb14ca87e6f95c14f56bc2593049" }
-signet-zenith = { git = "https://github.com/init4tech/signet-sdk", rev = "b8251ff0fec7cb14ca87e6f95c14f56bc2593049" }
+signet-constants = { git = "https://github.com/init4tech/signet-sdk", rev = "ba5894f6fac35299d495ae115cd465a1f0791637" }
```

Comment:
note that technically there's a version mismatch here: `bin-base` will pull `signet-constants` from commit `bd183b627dcb0eb682da801093b13f1f8311446b` while this dep will be exclusively using this commit. This shouldn't cause problems right now, but it is something to be aware of. We could bump signet-constants as a patch release to make sure it's at the commit? @prestwich